cluster-up/cluster/kind-k8s-sriov-1.17.0/certcreator/certsecret.go

```diff
@@ -57,7 +57,7 @@ func generate(hookName, namespace string) ([]byte, []byte, error) {
 	if err != nil {
 		return nil, nil, fmt.Errorf("failed to generate self-signed certificate: %v", err)
 	}
-	log.Printf("Self-Signed certificate created sucessfully for CN %s", certConfig.CommonName)
+	log.Printf("Self-Signed certificate created successfully for CN %s", certConfig.CommonName)
```

Comment:
Can you please port this change over to [kubevirtci cluster-up](https://github.com/kubevirt/kubevirtci/tree/master/cluster-up)? It will be overwritten on next kubevirtci bump as the `cluster-up` folder is vendored in from [kubevirtci cluster-up](https://github.com/kubevirt/kubevirtci/tree/master/cluster-up) and your change will then get deleted.

Reply:
Thanks, updated, PTAL.